third_party/packages/ansi_up/lib/ansi_up.dart

```diff
@@ -2,76 +2,306 @@
+      //    this._osc_regex = rgx(__makeTemplateObject(["\n                        ^                           # beginning of line\n                                                    #\n                        \u001B]8;                    # OSC Hyperlink\n                        [ -:<-~]*       # params (excluding ;)\n                        ;                           # end of params\n                        ([!-~]{0,512})        # URL capture\n                        (?:                         # ST\n                          (?:\u001B\\)                  # ESC                           |                           # alternate\n                          (?:\u0007)                    # BEL (what xterm did)\n                        )\n                        ([!-~]+)              # TEXT capture\n                        \u001B]8;;                   # OSC Hyperlink End\n                        (?:                         # ST\n                          (?:\u001B\\)                  # ESC                           |                           # alternate\n                          (?:\u0007)                    # BEL (what xterm did)\n                        )\n                    "], ["\n                        ^                           # beginning of line\n                                                    #\n                        \\x1b\\]8;                    # OSC Hyperlink\n                        [\\x20-\\x3a\\x3c-\\x7e]*       # params (excluding ;)\n                        ;                           # end of params\n                        ([\\x21-\\x7e]{0,512})        # URL capture\n                        (?:                         # ST\n                          (?:\\x1b\\\\)                  # ESC \\\n                          |                           # alternate\n                          (?:\\x07)                    # BEL (what xterm did)\n                        )\n                        ([\\x21-\\x7e]+)              # TEXT capture\n                        \\x1b\\]8;;                   # OSC Hyperlink End\n                        (?:                         # ST\n                          (?:\\x1b\\\\)                  # ESC \\\n                          |                           # alternate\n                          (?:\\x07)                    # BEL (what xterm did)\n                        )\n                    "]));
+      //  }
+      //  var match = this._buffer.match(this._osc_regex);
+      //  if (match === null) {
+      //    pkt.kind = PacketKind.ESC;
+      //    pkt.text = this._buffer.slice(0, 1);
+      //    this._buffer = this._buffer.slice(1);
+      //    return pkt;
+      //  }
+      //  pkt.kind = PacketKind.OSCURL;
+      //  pkt.url = match[1];
+      //  pkt.text = match[2];
+      //  var rpos = match[0].length;
+      //  this._buffer = this._buffer.slice(rpos);
+      //  return pkt;
+      //}
+    }
+    return pkt;
+  }
+
+  void _appendBuffer(String text) {
```

Comment:
remove this buffer and use
_buffer.write(text)
once you switch to a StringBuffer